src/main/java/edu/utdallas/davisbase/representation/WhereExpression.java

```diff
@@ -0,0 +1,30 @@
+package edu.utdallas.davisbase.representation;
+
+import net.sf.jsqlparser.expression.Expression;
+
+public class WhereExpression {
```

Comment:
This class is missing the following getter methods:

- `public String getExpression()`
- `public boolean isNot()`
- `public String getColumn()`
- `public Operator getOperator()` (or whatever ((structured)) class you want to use to represent the valid DavisBase comparison operators)
- `public Expression getValue()` (or whatever ((structured)) you class you want to use to represent a literal value expression)